api/src/main/java/io/opentelemetry/trace/DefaultTracer.java

```diff
@@ -54,7 +54,7 @@ public Span startSpan() {
     @Override
     public NoopSpanBuilder setParent(Context context) {
       Objects.requireNonNull(context, "context");
-      spanContext = TracingContextUtils.getSpan(context).getContext();
+      spanContext = Span.fromContext(context).getContext();
```

Comment:
Since it's like `SpanContext` won't actually be renamed, `getContext()` desparately needs to be renamed to `getSpanContext()`.